### DISCUSSION TITLE
[Question]: Dependency preview and recordings

### DISCUSSION THREAD

**Discussion by vigeland** [blakeblackshear/frigate#13117] [Ask A Question] [answered]
### What is your question:

I have switched off all 4 modes. Recording, snapshot, dectect and motion. Preview video is still generated and can be viewed. 
How can I ensure that no recordings are created? I send the settings via Mqtt.

**Comment by NickM-27** [comment, marked as answer]
The preview will be deleted at the next recording cleanup. We will ensure previews are not created in a future version.
